src/main/java/org/broadinstitute/consent/http/mail/freemarker/FreeMarkerTemplateHelper.java

```diff
@@ -208,4 +215,18 @@ private Writer generateDisabledDatasetsTemplate(String user, List<String> datase
     return out;
   }
 
+  private Writer generateNewDAARequestTemplate(
+      String signingOfficialUserName,
+      String userName,
+      String daaName,
+      String serverUrl,
+      Template temp
+  ) throws IOException, TemplateException {
+    NewDaaRequestModel model = new NewDaaRequestModel(serverUrl, daaName, userName,
```

Comment:
[General Consent question]

Just curious, does this repository have linting enabled (e.g. via Spotless)?  I ask because I've seen some formatting inconsistencies that Spotless may have picked up if it were running.

Reply:
No, I don't think we do.

Reply:
We have code styles and IntelliJ built-in rules checked into the repository, but we don't have a separate Spotless task.